config/express.js

```diff
@@ -1,49 +1,51 @@
 var passport = require('passport'),
   RedditStrategy = require('passport-reddit').Strategy;
 
-var verifyHandler = function(token, tokenSecret, profile, done) {
-  process.nextTick(function() {
-    User.findOne({uid: profile.id}, function(err, user) {
-      Reddit.getBothFlairs(tokenSecret, profile.name, function (flair1, flair2) {
-        if (user) {
-          if (user.banned) {
-            return done("You are banned from FAPP", user);
-          }
-          user.redToken = tokenSecret;
-          user.flair = {ptrades: flair1, svex: flair2};
-          user.save(function (err) {
-            if (!err) {
-              return done(null, user);
-            } else {
-              return done(null, user);
+module.exports = {
+  verifyHandler: function(token, tokenSecret, profile, done) {
+    process.nextTick(function() {
+      User.findOne({uid: profile.id}, function(err, user) {
+        Reddit.getBothFlairs(sails.config.reddit.adminRefreshToken, profile.name, function (flair1, flair2) {
```

Comment:
That would also work, though the callback for a `RedditStrategy` will always have only those four parameters if we're using passport-reddit. To pass in the object, we could do something like this:

    var verifyHandler = function (adminToken, token, tokenSecret, profile, done) {
        ...
    }
    
    module.exports.http = {
        var callWithToken = function (token, tokenSecret, profile, done) {
            verifyHandler(sails.config.reddit.adminRefreshToken, token, tokenSecret, profile, done);
        };
        ...
        passport.use(new RedditStrategy({
            key1: value1,
            key2: value2,
            ...
        }, callWithToken));
    };

Would that be better than putting all of verifyHandler in module.exports?